### PR TITLE
Implement missing cheer emote tooltip

### LINKED ISSUE #2434: Cheer emotes don't have feature parity with BTTV emotes.
**Describe the bug**

Cheer emotes (Cheer1, Cheer10, pajaCheer1, etc.) cannot be zoomed in. You cannot open links to the GIF.

**To reproduce**

 1. Find a channel with people cheering.
 2. Hover over the cheer emote and see that it doesn't have a preview.
 3. Right click the emote and see that you cannot open the link in a browser.

**Screenshots**

N/A

**Chatterino version**

Chatterino + Dankerino patches 763acf9a based on upstream c2782666a66810ef6b973ce543626ad80d07e352

**Operating system**

Ubuntu 20.04.1 LTS (Focal Fossa)

**Additional information**

### PR DESCRIPTION
**Pull request by jammehcow**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

This PR fixes/adds the ability to hover over a cheer emote and display a tooltip. This is because no tooltip is set on the cheer emotes when they are parsed/created - failing a check in [ChannelView.cpp#L1406](https://github.com/Chatterino/chatterino2/blob/master/src/widgets/helper/ChannelView.cpp#L1406) where a tooltip must be present in order to show.

This aims to fix #2434

**Steps to reproduce:**
  1. Find/create a cheer message containing a cheer emote
  2. Hover over the emote

**Expected result:**
Emote tooltip displays with a larger version of the emote as well as the text value of the emote

**Actual result (prior to this PR):**
No tooltip is displayed.

_NOTE: I'm not a proficient C++ dev, nor do I have much experience with QT so if it looks weird just comment on the line with an example of how I can do better._

### CODE REVIEW

**Comment by jammehcow** [comment]
Example of expected behavior from this PR.
Chatterino2_CheerEmote_ToolTip_Shown

File: src/providers/twitch/TwitchChannel.cpp [hunk 1]
```diff
@@ -909,22 +909,25 @@ void TwitchChannel::refreshCheerEmotes()
                     // We will continue to do so for now since we haven't had to
                     // solve that anywhere else
 
+                    // Combine the prefix (e.g. BibleThump) with the tier (1, 100 etc.)
+                    auto emoteTooltip =
+                        set.prefix + tier.id + "<br/>Twitch Cheer Emote";
```
Comment (zneix) [inline src/providers/twitch/TwitchChannel.cpp:914]: small nitpick, just to keep consistency with all the other `<br>`'s in the codebase
```suggestion
                        set.prefix + tier.id + "<br>Twitch Cheer Emote";
```

**Review by zneix — CHANGES_REQUESTED** [review]

File: src/providers/twitch/TwitchChannel.cpp (hunk 1 above)
Comment (jammehcow) [reply src/providers/twitch/TwitchChannel.cpp]: Resolved.

**Review by zneix — APPROVED** [review]

**Comment by pajlada** [comment]
I have only taken a look at the code but it looks good! I will test this out this weekend and get full approval at that point. Thank you!

**Review by pajlada — APPROVED** [review]
Looks good to me :+1:

**Comment by jammehcow** [comment]
Cheers.

**Pull request merged**